### DISCUSSION TITLE
TiffLibrary no longer plays nice with 2.1.x

### DISCUSSION THREAD

**Discussion by jhwheuer** [SixLabors/ImageSharp#2195] [Q&A] [answered]
Hi Jim, take a look at this thread... The Bridge to TiffLibrary appears broken after ImageSharp 2.0... https://github.com/yigolden/TiffLibrary/issues/14

I have taken TIFF handling off my apps in response.

**Comment by antonfirsov** [comment, marked as answer]
The exception seems to be thrown from `TiffLibrary.ImageSharpAdapter.TiffDecoder` & the information in the issue is hardly actionable for us unless we debug into TiffLibrary ourselves. Would be helpful if someone could get more info to understand what is broken, it might be something about common NuGet dependencies.

**Reply by JimBobSquarePants** [reply]
I'm struggling to see what this has to do with us or why it was brought to our attention. That's not our concern to maintain and we have our own Tiff encoders and decoders.

**Reply by jhwheuer** [reply]
Hi Jim, my spidy sense is tingling when, in a sub-version update, something breaks that hooks into another library... Maybe something else might be broken without being realized... It was definitely the ImageSharp sub-version change that did the deed... 

Just housekeeping, I guess.

**Reply by JimBobSquarePants** [reply]
Version 2.1 had [documented breaking changes](https://github.com/SixLabors/ImageSharp/releases/tag/v2.1.0) that affected our decoder API.

 Again though, their interoperability is none of our concern. We can follow any versioning strategy we choose and the changes did not warrant a major release for us. Even Microsoft make [breaking changes with minor versions](https://docs.microsoft.com/en-us/dotnet/core/compatibility/3.1) so it's not unheard of.

**Reply by jhwheuer** [reply]
Hey Jim, I did not mean any offense... Will knock on the TiffLibrary's door :-)

**Reply by JimBobSquarePants** [reply]
No worries mate, just wanted to make our position clear. It looks like they just need to update their dependency.

**Reply by jhwheuer** [reply]
:-) Just let them know in my original thread.

**Comment by brianpopow** [comment]
@jhwheuer out of curiosity: What Tiff features does this [TiffLibrary](https://github.com/yigolden/TiffLibrary/issues/14) support which ImageSharp does not yet support?

**Reply by jhwheuer** [reply]
Oh, when I started working on my App Just-Snips a while back, the TIFF Meta Data support in ImageSharp was a bit sketchy... 
I saw that you worked on that, so when I re-activate TIFF support I'll take a look if I can use that instead of the kludge...
